app/plugin.go

```diff
@@ -111,8 +109,6 @@ func (a *App) SyncPluginsActiveState() {
 				}
 			}
 		}
-	} else { // If plugins are disabled, shutdown plugins.
-		pluginsEnvironment.Shutdown()
```

Comment:
I don't know why this was here in the first place. This should be done in `App.ShutDownPlugins()`